Data/Sys/GameSettings/GHAE08.ini

```diff
@@ -0,0 +1,13 @@
+# GHAE08 - Resident Evil 2
+
+[OnFrame]
+# Skip memset call that is followed by dcbi, simulating dcache behavior
```

Comment:
Maybe I'm splitting hairs here, but it doesn't really "simulate" the dcache behaviour. 

It fixes a bug that causes the original game to rely on a correctly implemented dcache,

Reply:
I used the word simulate because it produces the same observable side effect as a functional data cache: the memset is canceled, so to speak. In that sense, it makes Dolphin deviate less from the behavior of the game on real hardware.

It can also be fairly described as a bug fix for the game, but whereas some bug fixes would cause a greater deviation from the game's behavior on real hardware, this one does the opposite.

Reply:
I don't like "simulate" because it insinuates the patched behaviour will be identical to unpatched code on real hardware. 

The unpatched code has theoretical edge cases where it might produce glitchy audio even on real hardware. There is quite a bit of code between the memset and the dcbi (they are in different functions). 
  
An interrupt might come along at the wrong moment and execute more code causing extra code to be executed, which might replace and commit some cachelines. Or an intermittent disk read error might cause a thread switch before retrying.

Reply:
How "roughly approximating dcache behavior"? Or would you prefer if the comment went into extra detail?

Reply:
"approximating" is a better word.

Though more detail is always better. Imagine you stumble across this .ini file knowing nothing about RE2 or the history of this bug and the pull requests. What details would such a reader want to know in a quick summary? 

For me, the key points are:

 * It was a bug in the original game
 * The bug causes background sounds to be zeroed during load
 * The bug was masked on real hardware by dcache
 * This fully fixes the bug. This patch should also work on real hardware
 * dolphin doesn't emulate dcache because the performance hit would be huge,.

Reply:
I believe most of these points are touched upon by the commit message, but I can put a summary in the .ini as well.

Reply:
Yeah, summary in .ini. 

Sometimes the whole git repo gets rearranged and it might be hard to track back those changes to a git commit or a PR

Reply:
Mind if I just use your summary, paraphrased slightly?

> Work around for a game bug that causes background sounds to be zeroed during load. The bug was masked on real hardware by dcache. This patch fully fixes the bug and should also work on real hardware. Dolphin doesn't emulate dcache because the performance hit would be huge.